requirements/install

```diff
@@ -0,0 +1,68 @@
+# Please make sure to cap all dependency versions, in order to avoid unwanted
+# functional and integration breaks caused by external code updates.
+#
+# These are general dependencies, so:
+#
+# * the ranges should be broad, and
+# * the number of them kept discreet.
+#
+# If a dependency seems trivial, please propose a pull request to remove it.
```

Comment:
```suggestion
# If a dependency seems redundant, please propose a pull request to remove it.
```

Reply:
I was meaning _unnecessary_. Maybe that's a better wording?

Reply:
Yep, good for me.